extensions/sigstore.md

```diff
@@ -0,0 +1,51 @@
+# Signature Extension: Sigstore
```

Comment:
Should we add a CODEOWNERS file so that a set of sigstore maintainers are notified when this file is updated?

Reply:
Sounds reasonable but if we're moving to a common list for multiple extensions, that could get spammy?